src/sentry/notifications/notifications/organization_request/invite_request.py

```diff
@@ -13,7 +13,7 @@
 @register()
 class InviteRequestNotification(AbstractInviteRequestNotification):
     analytics_event = "invite_request.sent"
-    referrer_base = "invite_request"
+    metrics_key = "invite-request"
```

Comment:
@mgaeta this is going to break a bunch of existing analytics as we are checking for `invite_request` as part of the referrer

Reply:
This is true for many of the other event types as well

Reply:
If updating the value for metrics_key is going to break analytics, it should not be changed.  If this is not conforming to a stylistic standard we have, then should be addressed separately.  As it is, I don't see why this change is needed for API-2511